lib/heroku/command/pg_backups.rb

```diff
@@ -628,7 +630,7 @@ def hpg_app_client(app_name)
   end
 
   def parse_schedule_time(time_str)
-    hour, tz = time_str.match(/([0-2][0-9]):00 (.*)/) && [ $1, $2 ]
+    hour, tz = time_str.match(/([0-2][0-9]):00 ?(.*)/) && [ $1, $2 ]
```

Comment:
I think a viable alternative would be using [Shellwords.shellsplit](http://ruby-doc.org/stdlib-2.0.0/libdoc/shellwords/rdoc/Shellwords.html#method-c-shellsplit). I can look into this if that seems like a good idea.